pyproject.toml

```diff
@@ -41,6 +41,7 @@ dependencies = [
   "types-PyYAML",
   "boto3",
   "boto3-stubs[essential]",
+  "moto",
```

Comment:
Is `moto` a runtime dependency? I.e., needed to use certain features of tsdat? Or is it just needed for development purposes? I ask bc it's already included as an optional dependency in the `[project.optional-dependencies]` --> `dev` section (ie. `pip install ".[dev]"` would include it)